content/en/agent/guide/azure-private-link.md

```diff
@@ -55,8 +55,21 @@ You can configure Azure Private Link to expose a private IP address for each Dat
    - For **Name**, enter `*`.
    - For **Type**, select **A - Address record**.
    - For **IP address**, enter the IP address you noted at the end of the previous section.
-   
+
    Select **OK** to finish.
+### Additional required steps for metrics and traces
+There are two Datadog Intake Services that are subdomains of the (`agent.`{{< region-param key="dd_site" code="true" >}}) domain. Because of this, the Private DNS zone is slightly different from other intakes.
```

Comment:
```suggestion
There are two Datadog Intake Services that are subdomains of the `agent.`{{< region-param key="dd_site" code="true" >}} domain. Because of this, the Private DNS zone is slightly different from other intakes.
```